### PR TITLE
build home screen and items

### PR DESCRIPTION
**Pull request by thanhtam123**
https://edu-redmine.framgia.vn/issues/392

### CODE REVIEW

**Comment by thanhtam123** [comment]
ready

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <LinearLayout
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_home.xml]: dùng constraint đi em

File: app/src/main/res/layout/item_categoty_movie.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    android:orientation="vertical"
+    android:padding="@dimen/padding">
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <TextView
+            android:id="@+id/text_item_name_category"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Popular"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_categoty_movie.xml]: hardcode

File: app/src/main/res/layout/item_categoty_movie.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            android:textColor="@color/white"
+            android:textSize="20sp" />
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_categoty_movie.xml]: hardcode

File: app/src/main/res/layout/item_genres.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="40dp"
+    android:background="@color/colorAccent"
+    android:gravity="center">
+
+    <TextView
+        android:id="@+id/text_name_genres"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Action"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_genres.xml]: hardcode

File: app/src/main/res/layout/item_genres.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        android:textColor="@color/white"
+        android:textSize="20sp"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_genres.xml]: hardcode

**Comment by thanhtam123** [comment]
ready

**Comment by thanhtam123** [comment]
ready

**Comment by DoanVanToan** [comment]
commented

**Comment by thanhtam123** [comment]
ready

File: app/src/main/res/layout/item_categoty_movie.xml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentRight="true"
+            android:background="@android:color/transparent"
+            android:src="@mipmap/ic_more_category"
+            android:textColor="@color/white" />
+    </RelativeLayout>
+
+    <android.support.v7.widget.RecyclerView
+        android:paddingLeft="4dp"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_categoty_movie.xml]: hardcode

File: app/src/main/res/menu/activity_home_drawer.xml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        <item
+            android:id="@+id/nav_favorites"
+            android:icon="@mipmap/ic_favorite"
+            android:title="Favorites" />
+        <item
+            android:id="@+id/nav_search"
+            android:icon="@mipmap/ic_search"
+            android:title="Slideshow" />
+        <item
+            android:id="@+id/nav_genre"
```
Comment (DoanVanToan) [inline app/src/main/res/menu/activity_home_drawer.xml]: hardcode

File: app/src/main/res/layout/fragment_home.xml [hunk 8]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_home.xml]: remove this constraint
ko cần constraint này

**Comment by DoanVanToan** [comment]
commented

**Comment by thanhtam123** [comment]
ready

File: app/src/main/res/values/colors.xml [hunk 9]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorPrimary">#1a0042</color>
+    <color name="colorPrimaryDark">#00001e</color>
+    <color name="colorAccent">#46286e</color>
+    <color name="colorStar">#ff9800</color>
```
Comment (DoanVanToan) [inline app/src/main/res/values/colors.xml]: color dat ten ttheo format
color_color_name
vi du
color_green
color_red
...
http://chir.ag/projects/name-that-color/
cach xem mau

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    <dimen name="item_movie_height_category">170dp</dimen>
+    <dimen name="item_text_genres_name">20sp</dimen>
+    <dimen name="text_size_category_name_title">20sp</dimen>
+    <dimen name="margin_4dp">4dp</dimen>
+    <dimen name="margin_8dp">8dp</dimen>
+    <dimen name="margin_0dp">0dp</dimen>
+    <dimen name="card_corner_radius">2dp</dimen>
+    <dimen name="card_elevation">3dp</dimen>
+
+</resources>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: add __end of file

**Comment by DoanVanToan** [comment]
commented

**Comment by thanhtam123** [comment]
ready

**Pull request merged**